src/tests-tokens/strict-mode-tests.ts

```diff
@@ -0,0 +1,266 @@
+import * as nock from 'nock';
```

Comment:
Another useful couple tests could be around the `performFetch` function error conditions when called by `getMetadataFromUri`. For example, exceeding `METADATA_FETCH_TIMEOUT_MS` and `METADATA_MAX_PAYLOAD_BYTE_SIZE`.